src/components/postCard/children/postCardStyles.js

```diff
@@ -118,4 +118,10 @@ export default EStyleSheet.create({
     textAlign: 'center',
     fontSize: 18,
   },
+  promotedText: {
+    fontSize: 14,
+    fontWeight: '300',
+    color: '$primaryDarkGray',
+    marginTop: 6,
```

Comment:
may be try adding negative bottom margin/padding here to make upper and lower spacing more fine tuned
<img width="325" alt="Screenshot 2023-12-18 at 15 38 19" src="https://github.com/ecency/ecency-mobile/assets/6298342/e0d3ef71-abcd-4169-91e0-9bcdf28fb5e2">